codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/extensions/EObjectTypeExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package io.vrap.codegen.languages.extensions
+
+import io.vrap.rmf.codegen.types.VrapNilType
+import org.eclipse.emf.ecore.EObject
+
+interface EObjectTypeExtensions : ExtensionsBase {
```

Comment:
What's the difference between this interface and the one above `EObjectExtensions`? Why do we need both?

Reply:
i think this one adds the nullability check, this is the main difference, it was just a refactoring so i m not sure whats the use case for this.

Reply:
i will leave only one with nullability check